data/brands/amenity/fuel.json

```diff
@@ -6383,6 +6383,19 @@
         "brand": "福懋加油站",
         "name": "福懋加油站"
       }
+    },
+    {
+      "displayName": "apollostation",
+      "locationSet": {"include": ["jp"]},
+      "tags": {
+        "amenity": "fuel",
+        "brand": "apollostation",
+        "brand:en": "apollostation",
+        "brand:ja": "アポロステーション",
+        "name": "apollostation",
```

Comment:
There are many other Japanese brands whose official names are written in the alphabet.
For example, the following items.

- [AOKI](https://nsi.guide/index.html?t=brands&k=shop&v=clothes&tt=AOKI)
- [TSUTAYA](https://nsi.guide/index.html?t=brands&tt=TSUTAYA)
- [WEGO](https://nsi.guide/index.html?t=brands&k=shop&v=clothes&tt=WEGO)

Therefore, I assume that the same rule applies to apollostation as these.

Also, if the name tag of Japanese brands must always be in Japanese, I think it would apply to the Viva Home issue (although Viva Home's official brand name is in Japanese anyway).

Reply:
I'm sorry to hear that the merge is complete. 

In "JA:Naming sample", "出光と昭和シェルは順次こちらに切り替え(Idemitsu and Showa Shell will switch to this one after another)" Anyway, how about deleting the items of "Idemitsu" and "Showa Shell" and incorporating them into the "matchNames" of apollostation?
```fuel sample
    {
      "displayName": "apollostation",
      "locationSet": {"include": ["jp"]},
      "matchNames": [
        "idemitsu","出光","ロイヤル・ダッチ・シェル", "昭和シェル", "昭和シェル石油"
      ],
      "tags": {
        "amenity": "fuel",
        "brand": "apollostation",
        "brand:en": "apollostation",
        "brand:ja": "アポロステーション",
        "brand:wikidata": "Q2216770",
        "name": "apollostation",
        "name:en": "apollostation",
        "name:ja": "アポロステーション"
      }
    }
```

Reply:
According to the company's announcement, "all Service Stations will be switched to apollostation by the end of 2023", so some stands(stations) have not yet been switched. Therefore, I think it is a little premature.